pkg/ocm/ocm.go

```diff
@@ -237,9 +239,57 @@ func (c *SdkClient) LimitedSupportExists(ls LimitedSupportReason, internalCluste
 	return false, nil
 }
 
+// isLimitedSupportReasonFlapping checks if a cluster has been put in limited support too many times
+// Returns true if the cluster has been put in limited support at least 2 times in the last 4 hours
```

Comment:
**question:** I wonder if this is a good value, is this based on the ticket/slack threads data? This value needs to catche the occurences where we have automation fighting the hive restarts.

Reply:
I reviewed again the past cases and upped it to 24 hours. I think it should cover it better now.